val.py

```diff
@@ -340,6 +340,12 @@ def run(
                                               confusion_matrix=confusion_matrix)
                 else:
                     save_one_txt(predn, save_conf, shape, file=save_dir / 'labels' / f'{path.stem}.txt')
+                    # Print results
+                    text = ''
+                    for c in predn[:, 5].unique():
+                        n = (predn[:, 5] == c).sum()  # detections per class
+                        text += f"{n} {names[int(c)]}{'s' * (n > 1)}, "  # add to string
```

Comment:
What is this supposed to log and can you give me an example?

Reply:
I put it in the description of the PR.
# of persons and licences per image